### PR TITLE
TAG 2650 - refresh groups roles

### PR DESCRIPTION
**Pull request by joaopgg**

### CODE REVIEW

File: components/angular-tomitribe-select/src/tomitribe-select.ts [hunk 1]
```diff
@@ -151,6 +153,26 @@ module tomitribe_select {
...
+            restrict: 'A',
+            require: 'uiSelect',
+            replace: false,
+            link: link,
+        };
+
+        function link(scope, element, attrs, uiSelectCtrl) {
+            scope.$on('uis:activate', ()=> {
+                //Check if we have the function defined  in tribe-select-fetch-on-open="function". Otherwise, check if the have the refresh attribute defined in ui-select
+                let refreshFn = attrs['tribeSelectFetchOnOpen'].indexOf('tribe-select-fetch-on-open') !== 0 ? attrs['tribeSelectFetchOnOpen'] : element.find('.ui-select-dropdown').attr('refresh');
```
Comment (Dexmaster) [inline components/angular-tomitribe-select/src/tomitribe-select.ts]: This is just wrong, please fix indexOf extravaganza, 
also if possible do not use element.find you can:
 - use attribute on ui-select-choices
 - require both if you need them uiSelectCtrl and uiSelectChoicesCtrl require: ['uiSelect', 'uiSelectChoices']
 - have uiSelectCtrl.refresh and refresh function from attrs there (attrs['refresh'] without tribeSelectFetchOnOpen function)

**Review by Dexmaster — CHANGES_REQUESTED** [review]
Requesting changes use refresh attribute please

File: components/angular-tomitribe-select/src/tomitribe-select.ts (hunk 1 above)
Comment (Dexmaster) [reply components/angular-tomitribe-select/src/tomitribe-select.ts]: Another solution you can just force trigger search change on activation which will do everything for you. Still use refresh attribute in choices for everything. Thank you.
https://github.com/angular-ui/ui-select/blob/3f097e0a717e40e6fc931b7655afcc8a3c575f91/src/uiSelectChoicesDirective.js#L63

**Comment by joaopgg** [comment]
Thanks for the comments @Dexmaster, must more simple with your suggestion

**Comment by Dexmaster** [comment]
+1

**Comment by pavelsmajda** [comment]
+1

**Pull request merged**